DEVELOPING.md

```diff
@@ -0,0 +1,67 @@
+# Cylc Rose Dev Docs
+
+Cylc Rose provides integration between Cylc and Rose.
+
+Its functionality is defined in the
+[cylc-rose proposal](https://github.com/cylc/cylc-admin/blob/master/docs/proposal-cylc-rose-installing-rose-configs.md).
+
+## Cylc Rose Options
+
+### Rose Config Files
+
+Rose suite configurations are Cylc workflows which contain a `rose-suite.conf`
+file at the top level. This file configures:
+
+* Template variables.
+* File installation.
+* Environment variables for workflow configuration.
+
+Rose suite configurations may additionally contain optional configuration files
+in the `opts/` directory. Note that optional configurations may be turned on by
+default via the `opt` configuration in the `rose-suite.conf` file.
+
+### The Options
+
+The Rose suite configuration may be extended or overridden via the Cylc Rose
+CLI options.
+
+Cylc Rose adds three options to Cylc Flow.
+
+* `-S` - template variables
+* `-D` - any arbitrary rose configuration (includes template variables)
+* `-O` - optional configs
+
+These options are currently hardcoded in the cylc-flow source code as we
+do not yet have the ability for Cylc plugins (such as cylc-rose) to inject
+options into cylc-flow commands.
+
+There is also the `ROSE_SUITE_OPT_CONF_KEYS` environment variable which compliments
+the `-O` option.
+
+### Storage
+
+Cylc Rose options passed in on the CLI are written to the
+`~/cylc-run/<workflow-id>/opt/rose-suite-cylc-install.conf` file by
+the `post_install` plugin.
+
+This preserves these options so that they are inherited by subsequent commands
+avoiding the need for users to remember which Cylc Rose options they have used
+and specify them with all future commands.
+
+### Reinstallation
+
+On reinstallation, additional Cylc Rose CLI options may be provided.
```

Comment:
The word "additional" suggests options in addition to `-D`, `-S` , `-O` already mentioned
```suggestion
On reinstallation, the Cylc Rose CLI options may be provided.
```

Reply:
On reinstallation you may provide additional `-D`, `-S`, `-O`, `$ROSE_SUITE_OPT_CONF_KEYS` arguments.

Reply:
With the context of the next sentence, I don't see any issues here.